frontend/src/router/index.js

```diff
@@ -101,6 +101,21 @@ const Message = () => import('@/views/apps/email/Message')
 
 Vue.use(Router)
 
+const validPathNames = [
```

Comment:
Looks good, except this change from #42 that has been resolved by @purplesmurf1998 and @AdrienKamran on the previous pull request.

Reply:
Yes, it seems that it was carried over. I have updated in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/44/commits/f7136c400971f2d84bd6c9d86b98e5c962d513a3)